skbio/diversity/beta/_base.py

```diff
@@ -13,28 +13,24 @@
 import numpy as np
 from scipy.spatial.distance import pdist
 
-from skbio.diversity.beta._unifrac import unweighted_unifrac, weighted_unifrac
+from skbio.diversity.beta._unifrac import (_unweighted_unifrac_pdist_f,
+                                           _weighted_unifrac_pdist_f)
+
 from skbio.stats.distance import DistanceMatrix
 from skbio.util._decorator import experimental, deprecated
 
 
-def _get_skbio_metrics():
```

Comment:
define a new function importable as ``skbio.diversity.get_beta_diversity_metrics`` which returns a list of the beta diversity metrics implemented in scikit-bio. The doc string should probably mention the SciPy metrics and where to find information on this (in SciPy's docs).